content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,156 @@ aliases:
-- Provides monitoring benefits.
-For details, see [Mixer policies and telemetry](/docs/reference/config/policy-and-telemetry/).
-
-For example, in A/B testing we often want to configure traffic routes based on
-percentages. With Istio, you can use a virtual service to specify a routing
-rule that sends 25% of requests to instances in the `v2` subset, and sends the
-remaining 75% of requests to instances in the `v1` subset. The following
-configuration accomplishes our example for the `reviews` service.
+-   [Introducing Istio Traffic Management](/docs/concepts/traffic-management/#introducing-istio-traffic-management):
+    Learn about how Istio helps you control the flow of traffic in your mesh
+    with simple, flexible configuration.
+
+-   [Network resilience and testing](/docs/concepts/traffic-management/#network-resilience-and-testing):
+    Learn about Istio's dynamic failure recovery features that you can configure
+    to test and build tolerance for failing nodes, and to prevent cascading
+    failures to other nodes.
+
+-   [Architecture](/docs/concepts/traffic-management/#architecture): Learn about
+    Pilot, Istio's core traffic management component and Envoy proxies and how
+    they enable service discovery and traffic control for services in the mesh.
+
+Istio’s traffic routing rules let you easily control the flow
+of traffic and API calls between services. Istio simplifies configuration of
+service-level properties like circuit breakers, timeouts, and retries, and makes
+it easy to set up important tasks like A/B testing, canary rollouts, and staged
+rollouts with percentage-based traffic splits. It also provides out-of-box
+failure recovery features, which, combined with Istio’s
+[observability](/docs/concepts/observability/) features, helps make your calls
+more reliable, and your network more robust.
+
+Istio’s traffic management model relies on the Envoy proxies that are deployed
+along with your services. All traffic that your mesh services send and receive
+(data plane traffic) is proxied through Envoy, making it easy to direct and
+control traffic around your mesh without making any changes to your services -
+your workload just needs to specify the destination service, and the Envoy
+proxies do the rest.
+
+If you’re interested in the details of how the features described in this guide
+work, you can find out more about Istio’s traffic management architecture in the
+[Architecture](#architecture) section at the end of this document. The rest of
+this guide introduces Istio’s traffic management features.
+
+## Introducing Istio Traffic Management
+
+In order to direct traffic around your mesh, Istio needs to know where all your
+endpoints are, and which services they belong to. To populate its own service
+registry, Istio connects to a service discovery system. For example, if you've
+installed Istio on Kubernetes, then by default Istio knows about all the
+services and endpoints in that cluster.
```

Comment:
"that cluster" ?

Maybe the sentence should start with `For example, if you've installed Istio on a Kubernetes cluster ...`

Reply:
Better, thanks!

Reply:
Add another line for Consul